boards/nxp/mimxrt1180_evk/mimxrt1180_evk_mimxrt1189_cm7.dts

```diff
@@ -27,10 +27,10 @@
 	current-speed = <115200>;
 };
 
-&systick {
+&lpadc1 {
 	status = "okay";
 };
 
-&gpt2 {
```

Comment:
Why are you deleting the `gpt2` enablement for cm7?

Reply:
Sorry, most likely I made an error when I manually resolved the conflict.